ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -47,12 +47,12 @@
   /**
```

Comment:
Should we change the description of VungleBannerAdapter, and add details that match. current design of n:1 for Banner requests to VungleBannerAdapter instances for any given Placement
`
/**
 * A helper class used to load and show Vungle banner ads using Google Mobile Ads SDK mediation.
 */
`